Advice

amyyto opened the pull request.

redrambles commented:
Hey @amyyto !

This is great - the only thing we are missing is a link to your advice page from the main index.html file, so that folks can find your words of wisdom, once your code is merged in. Students will typically add their link to the top or the bottom of that long list. :)

Once you add your link, add and commit your changes and then push your branch up once more, which will automatically update your pull request. Then we should be good to merge in your code!

Let us know if you have any questions!

Cheers,
Ann

amyyto commented:
Hi @redrambles !! Thanks so much. I totally forgot to add/commit the main index.html after I added my line. I added it to the bottom of the page and pushed my branch up!

redrambles commented:
Hey @amyyto !

Sorry about the delay! Thanks for making those changes, your code is all merged in! :) 🎉 

Cheers,
Ann :)
